SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -274,6 +274,14 @@ void USpatialReceiver::OnAddComponent(const Worker_AddComponentOp& Op)
 		return;
 	}
 
+	// UNR-3066: Queue owner-only components until we have authority
+	if (ClassInfoManager->GetCategoryByComponentId(Op.data.component_id) == SCHEMA_OwnerOnly &&
```

Comment:
With result types, I believe we're guaranteed to only receive the owner only component when we're auth over the client end point. In which case, if we're receiving the data/update, we know we should apply it. Perhaps the auth check needs to be changed.

Reply:
I like it, but it could be slightly unintuitive for an Unreal dev to receive e.g. an OnRep for an OwnerOnly property and they would not be owned yet.